kube/app/ingress-internal.yml

```diff
@@ -18,6 +18,9 @@ spec:
         - {{ .APP_NAME }}.internal.uat.sas-notprod.homeoffice.gov.uk
       {{ else if eq .KUBE_NAMESPACE .STG_ENV }}
         - {{ .APP_NAME }}.internal.stg.sas.homeoffice.gov.uk
+      {{ end }}
+      {{ if eq .KUBE_NAMESPACE .BRANCH_ENV }}
+      secretName: branch-tls-internal
       {{ else }}
       secretName: {{ .APP_NAME }}-internal
```

Comment:
UAT env has secret named `ingress-internal`

Reply:
Hi @dk4g , I have renamed the secret in kubernetes namespace to use fmr-internal. 70 percent of projects are using {{app.name}}-internal and 30% of projects using ingress-internal.